src/utils.js

```diff
@@ -571,8 +573,8 @@ const htmlToText = (html) => {
 /**
  * Creates a standardized debug info from request and response. This info is usually added to dataset under the hidden `#debug` field.
  *
- * @param {Object} request [Apify.Request](https://sdk.apify.com/docs/api/request) object.
- * @param {Object} [response]
+ * @param {Request|RequestOptions} request [Apify.Request](https://sdk.apify.com/docs/api/request) object.
+ * @param {PuppeteerResponse|ServerResponse} [response]
```

Comment:
`IncomingMessage`, not `ServerResponse`.

Reply:
Done.